.semaphore/semaphore.yml

```diff
@@ -9,44 +9,37 @@ agent:
     type: e1-standard-2
     os_image: ubuntu1804
 
+global_job_config:
```

Comment:
we discussed the fact that none of the blocks (other than "Build") execute a `cache store`, however, currently all other blocks depend (either directly or indirectly) upon the "Build" block.
@Brian-McM - wdyt about adding a comment connecting the `cache store` commands in "Build" block to the prologue's actions, if you think it's helpful.

Reply:
I think that's a good idea, I'll add the comment.